orchestra/project_api/api.py

```diff
@@ -58,8 +73,10 @@ def get_workflow_steps(workflow_slug):
             continue
 
         already_added.add(current_node)
-        steps.append((current_node,
-                      workflow.get_step(current_node).description))
+        current_step = workflow.get_step(current_node)
+        steps.append({'slug': current_node,
```

Comment:
changed API to dict of step information rather than a tuple, since we went from a pair to a triple